### DISCUSSION TITLE
Authentication Failure Error when starting app

### DISCUSSION THREAD

**Discussion by simoesmiguel** [KRTirtho/spotube#1473] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I did install the dmg application on my Apple M1 Pro, listened some music and it worked properly for 2 days. 
After that when I start the app I always receive the same Authentication Failure Error.

<img width="777" alt="image" src="https://github.com/KRTirtho/spotube/assets/38257915/60500d57-5b0b-4f7c-adaa-3237635dcf1b">

### Expected Behavior

No error should be shown

### Steps to reproduce

* Install the dmg application.
* After quitting and opening the app multiple times the error should be shown 

### Operating System

macOS

### Spotube version

Version 3.5.0 (29)

### Installation source

Website (spotube.netlify.app) or (spotube.krtirtho.dev)

### Self grab

- [ ] I'm ready to work on this issue!

**Comment by Surechun** [comment]
me too, but  the Spotify was work fine.

**Comment by KRTirtho** [comment]
You've to logout and relogin to fix this issue. Sometimes, Spotify can revoke the `sp_dc` cookie, so you might even need to change the password to get it working again.

**Comment by KRTirtho** [comment]
Also it's a duplicate of #1268
